apps/backend/src/main.ts

```diff
@@ -4,15 +4,22 @@
  */
 
 import { Logger } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
 import { NestFactory } from '@nestjs/core';
+/*Classes */
+import { BackendConfig } from 'common-backend-models';
 
+/* Modules */
 import { AppModule } from './app/app.module';
 
 async function bootstrap() {
 	const app = await NestFactory.create(AppModule);
+
+	const configService = app.get(ConfigService<BackendConfig>);
+
 	const globalPrefix = 'api';
 	app.setGlobalPrefix(globalPrefix);
-	const port = process.env.PORT || 3000;
+	const port = configService.get('app', { infer: true }).port || 3333;
```

Comment:
Please decide whether you are using port 3333 or 3000 as default, currently main.ts and validation.ts use different fallback values

Reply:
As i am already setting a default, i will remove this